cmd/list.go

```diff
@@ -163,3 +165,25 @@ func printListContainerResponse(lsResponse *common.ListContainerResponse) {
 		glcm.Info(lsResponse.Blobs[index])
 	}
 }
+
+func ByteSizeToString(size int) string {
```

Comment:
we can make it lower-case since it doesn't need to be exported

Reply:
No longer exported as of [9cc9d04](https://github.com/Azure/azure-storage-azcopy/pull/341/commits/9cc9d04ee8a8bc98aee541a17413c42e0f359cfe)